.github/workflows/test.yml

```diff
@@ -13,6 +13,8 @@ jobs:
       - uses: actions/checkout@v3
       - run: |
           npm install
+      - run: |
+          npm install -g nx
```

Comment:
I think the installation of nx should be done on the other job, under the `test.steps` section, good catch tho! 👍 
Can you move it?